gobblin-runtime/build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
   compile externalDependency.gson
   compile externalDependency.guava
   compile externalDependency.guice
+  compile externalDependency.helix
```

Comment:
Hmm. I would avoid Helix dependency in gobblin-runtime.

Reply:
Can we move FlowStatusGenerator to gobblin-service module?

Reply:
I did try a lot of different locations for these things... the problem with doing that is that `FlowExecutionResource` needs `FlowStatusGenerator` as well, and I can't add gobblin-service as a dependency because gobblin-service depends on gobblin-restli.

Maybe I can try moving `FlowStatusGenerator` to gobblin-restli, that seems like it should be possible.

Reply:
Moved the killflow logic to flowExecutionResource so it's not necessary.